ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -0,0 +1,10 @@
+{# eos - dot1x global #}
+{% if dot1x.system_auth_control is arista.avd.defined(true) %}
```

Comment:
since this areas will most likely grow, I think we should add an outer `if dot1x is arista.avd.defined` to prevent having to run through all of these checks for every device.